includes/reports/data/taxes/class-tax-collected-by-location-list-table.php

```diff
@@ -145,12 +145,24 @@ public function get_data() {
 				? $wpdb->prepare( ' AND region = %s', esc_sql( $tax_rate->description ) )
 				: '';
 
-			$results = $wpdb->get_results( $wpdb->prepare( "
-				SELECT SUM(tax) as tax, SUM(total) as total, country, region
-				FROM {$wpdb->edd_orders}
-				INNER JOIN {$wpdb->edd_order_addresses} ON {$wpdb->edd_order_addresses}.order_id = {$wpdb->edd_orders}.id
-				WHERE {$wpdb->edd_order_addresses}.country = %s {$region} {$date_query}
-			", esc_sql( $tax_rate->name ) ), ARRAY_A );
+			$results;
+
+			if ( $location !== __( 'Global Rate', 'easy-digital-downloads' ) ) {
```

Comment:
Again we shouldn't be doing logical comparisons against a translateable value, and this would also need to be flipped as a yoda conditional.